app/views/books/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if books.any? %>
+  <ul class="books">
+    <% books.each do |book| %>
+      <li><%= book.title %></li>
+    <% end %>
+  </ul>
+<% else %>
+  You haven't added any books yet.
```

Comment:
Should we internationalize all text? https://robots.thoughtbot.com/better-tests-through-internationalization